docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+
+services:
+###> doctrine/doctrine-bundle ###
+  database:
+    image: postgres:${POSTGRES_VERSION:-13}-alpine
```

Comment:
This is not matching our [`.env`](https://github.com/sulu/skeleton/blob/8cd9b4ca22318006e3e3ae7bfe4460e6e1d3226c/.env#L35) file so we should update the docker-compose or our [to the symfony `.env`](https://github.com/symfony/recipes/blob/cb1cb50ace8fe990b60a7ea9e8f840cd6c30cbfb/doctrine/doctrine-bundle/2.4/manifest.json#L15) file to use postgresql.

/cc @sulu/core-team

Reply:
MySQL config could look like this: https://github.com/sulu/skeleton/pull/145